nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -61,26 +68,33 @@ class ParametersForm(BootstrapMixin, forms.Form):
         )
         mount_point = forms.CharField(
             required=False,
-            help_text=f"The path where the secret engine was mounted on (Default: <code>{DEFAULT_MOUNT_POINT}</code>)",
-            initial=DEFAULT_MOUNT_POINT,
+            help_text="The path where the secret engine was mounted on (Default: <code>secret</code>)",
```

Comment:
I have to make this support both old and new, but with you being able to have multiple configs, there is no way for me to show the DEFAULT_MOUNT_POINT for each configuration. Instead, we go with the original hard-coded default. I should probably add a note that these two fields are not meant to be user-configurable. They are there to allow the user to override, should they know what they are doing. Otherwise, the configuration that was configured by the Nautobot Admin would set this per configuration. But if neither are set, the default would be `secret`.